frontend/hub/collections/CollectionPage/CollectionContents.tsx

```diff
@@ -1,7 +1,226 @@
 import { useTranslation } from 'react-i18next';
-import { DetailInfo } from '../../../../framework/components/DetailInfo';
+import { useGet } from '../../../common/crud/useGet';
+import { pulpAPI } from '../../api/formatPath';
+import { useOutletContext } from 'react-router-dom';
+import { CollectionVersionSearch } from '../Collection';
+import { HubError } from '../../common/HubError';
+import { PulpItemsResponse } from '../../usePulpView';
+import { SearchInput } from '@patternfly/react-core';
+import { useSearchParams } from 'react-router-dom';
+import { ToolbarItem } from '@patternfly/react-core';
+import styled from 'styled-components';
+import { LoadingPage, PageTable } from '../../../../framework';
+import { PageSection } from '@patternfly/react-core';
+import { useEffect } from 'react';
+import { useRef } from 'react';
+import { EmptyStateNoData } from '../../../../framework/components/EmptyStateNoData';
+import { Link } from 'react-router-dom';
+import { useMemo } from 'react';
+import { ITableColumn } from '../../../../framework';
 
 export function CollectionContents() {
   const { t } = useTranslation();
-  return <DetailInfo title={t('This page is under construction')} />;
+  const { collection } = useOutletContext<{ collection: CollectionVersionSearch }>();
```

Comment:
It is rendering the `Showing` like 

![image](https://github.com/ansible/ansible-ui/assets/9053044/fa331d0d-8b6c-4fc7-bd6a-81ddf378e6ad)


From the old UI 

![image](https://github.com/ansible/ansible-ui/assets/9053044/49486083-5736-48b7-9780-3d20e0f47589)

Reply:
Any collection the problem appears as the same. We talked and @MilanPospisil is taking a look at this issue.